Exercise/src/tests.cpp

```diff
@@ -0,0 +1,42 @@
+#include <gtest/gtest.h>
+
+#include "my_string.h"
+class CMyStringFixture : public ::testing::Test
+{
+protected:
+	Exercise_1::CMyString myString;
+	const size_t sTestDataLength{17};
```

Comment:
define number 17 as it looks like a magic number

Reply:
Fixed